src/main/java/org/spongepowered/mod/mixin/core/entity/player/MixinEntityPlayer.java

```diff
@@ -24,21 +24,38 @@
  */
 package org.spongepowered.mod.mixin.core.entity.player;
 
+import org.spongepowered.api.Sponge;
 import org.spongepowered.common.mixin.core.entity.living.MixinEntityLivingBase;
+import org.spongepowered.common.util.VecHelper;
+
+import java.util.Optional;
 
 import org.spongepowered.asm.mixin.Shadow;
 import net.minecraft.entity.player.InventoryPlayer;
 import org.spongepowered.asm.mixin.Overwrite;
 import net.minecraft.entity.player.EntityPlayer;
+import org.spongepowered.api.entity.Transform;
+import org.spongepowered.api.event.SpongeEventFactory;
+import org.spongepowered.api.event.action.SleepingEvent;
+import org.spongepowered.api.event.cause.Cause;
+import org.spongepowered.api.world.World;
 import org.spongepowered.asm.mixin.Mixin;
+import com.flowpowered.math.vector.Vector3d;
+import net.minecraft.block.state.IBlockState;
 import net.minecraft.entity.item.EntityItem;
 import net.minecraft.item.ItemStack;
+import net.minecraft.util.BlockPos;
 
 @Mixin(EntityPlayer.class)
 public abstract class MixinEntityPlayer extends MixinEntityLivingBase {
 
     @Shadow public InventoryPlayer inventory;
+    @Shadow public BlockPos playerLocation;
+    @Shadow protected boolean sleeping;
+    @Shadow private int sleepTimer;
     @Shadow public abstract EntityItem dropItem(ItemStack droppedItem, boolean dropAround, boolean traceItem);
+    @Shadow public abstract void setSize(float width, float height);
```

Comment:
setSize is not part of the target class EntityPlayer but rather Entity so this needs to be removed.